docs/getting-started-guides/scratch.md

```diff
@@ -782,6 +745,48 @@ Template for controller manager pod:
 
 ```
 
+Flags to consider using with controller manager:
+ - `--cluster-name=$CLUSTER_NAME`
```

Comment:
The blank line does matter now, as we change to redcarpet as our markdown engine (https://github.com/GoogleCloudPlatform/kubernetes/pull/11584).

But my comment still holds, we still need the extra spaces before sub lists.